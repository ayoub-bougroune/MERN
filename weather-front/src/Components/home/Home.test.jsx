import React from 'react';
import renderer from 'react-test-renderer';
import Home from './index';

test('The Home renders correctly', () => {
  const home = renderer
    .create(<Home />);
  const tree = home.toJSON();
  expect(tree).toMatchSnapshot();
});