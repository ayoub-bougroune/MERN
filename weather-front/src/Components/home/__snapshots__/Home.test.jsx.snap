// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Home renders correctly 1`] = `null`;
